### PR TITLE
Right-to-left layout support (#200)

### PR DESCRIPTION
**Pull request by edenman**
@rotemlevi does this look correct?
screenshot_2015-04-28-00-03-00

@JakeWharton

### CODE REVIEW

**Comment by rotemlevi** [comment]
perfect !! thank you very much.

2015-04-28 10:09 GMT+03:00 Eric Denman <notifications@github.com>:

File: library/src/com/squareup/timessquare/MonthView.java [hunk 1]
```diff
@@ -57,6 +60,21 @@ public static MonthView create(ViewGroup parent, LayoutInflater inflater,
...
 
+  private static int getDayOfWeek(int firstDayOfWeek, int offset, boolean isRtl) {
+    int dayOfWeek = firstDayOfWeek + offset;
+    if (isRtl) {
+      return 8 - dayOfWeek;
+    }
+    return dayOfWeek;
+  }
+
+  private static boolean isRTL(Locale locale) {
```
Comment (JakeWharton) [inline library/src/com/squareup/timessquare/MonthView.java]: `isRtl`, acronyms always get camel cased to eliminate ambiguity: https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s5.3-camel-case

File: sample/src/com/squareup/timessquare/sample/SampleTimesSquareActivity.java [hunk 2]
```diff
@@ -55,14 +52,17 @@ private void initButtonListeners(final Calendar nextYear, final Calendar lastYea
...
-      @Override
-      public void onClick(View v) {
+      @Override public void onClick(View v) {
         single.setEnabled(false);
         multi.setEnabled(true);
         range.setEnabled(true);
         displayOnly.setEnabled(true);
+        decorator.setEnabled(true);
 
+        calendar.setDecorators(new ArrayList<CalendarCellDecorator>());
```
Comment (JakeWharton) [inline sample/src/com/squareup/timessquare/sample/SampleTimesSquareActivity.java]: `Collections.emptyList()`

**Comment by JakeWharton** [comment]
LGTM

**Comment by edenman** [comment]
Released in 1.6.2

**Pull request merged**